brandingbrand/flagship
# feat(fscomponents): add knobs to alert stories

**Description**
Added knobs to alert stories.

Because the Alert component is not rendered until the Button component in the story is pressed - knobs will not appear until the Alert component is rendered. I attempted to solve for this problem in several ways, but barring modifying the Alert or Button components themselves, there doesn’t seem to be a clean way to achieve knobs appearing before the Alert is rendered. I am open to suggestions if this is not sufficient.

**Testing**
- Pull down, `yarn run dev:storybook`
- Click Button to show Alert, modify the Alert knobs and click Button again to see knob changes
